### DISCUSSION TITLE
magit-display-buffer-fullcolumn-most-v1 deletes dedicated windows and does not restore them

### DISCUSSION THREAD

**Discussion by aaronjensen** [magit/magit#4687] [Support] [answered]
If you run `M-x magit-status` with `magit-display-buffer-function` set to `magit-display-buffer-fullcolumn-most-v1` in a window that is dedicated, the `delete-other-windows-vertically` invocation will delete the dedicated window. Then, when you quit `magit-status`, the dedicated window is not restored for some reason.

**Comment by tarsius** [comment, marked as answer]
Even with `(setq magit-bury-buffer-function #'magit-restore-window-configuration)`?

**Reply by kyleam** [reply]
Hmm, I tried quickly with

```elisp
(setq magit-display-buffer-function #'magit-display-buffer-fullcolumn-most-v1)
(setq magit-bury-buffer-function #'magit-restore-window-configuration)
```

and an otherwise vanilla config, and the window is restored on my end.

**Reply by aaronjensen** [reply]
For some reason I thought that behavior was the default. This corrects it, thank you!
